temporian/implementation/pandas/operators/sum.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Implementation for the sum operator."""
+
+from temporian.implementation.pandas.data.event import PandasEvent
+from temporian.implementation.pandas.operators.base import PandasOperator
+
+RESOLUTIONS = ["PER_FEATURE_IDX", "PER_FEATURE_NAME"]
+
+
+class PandasSumOperator(PandasOperator):
+    def __call__(
+        self, event_1: PandasEvent, event_2: PandasEvent, resolution: str = None
+    ) -> PandasEvent:
+        """Sum two EventSequences.
+
+        Args:
+            event_1: First EventSequence.
+            event_2: Second EventSequence.
+            resolution: PER_FEATURE_IDX -> Sum is done to each feature index wise.
+                        PER_FEATURE_NAME (Not implemented yet) -> Sum is done to each feature name wise.
```

Comment:
Comments and docstrings shouldn't go over the character limit for each line if possible